src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../patternfly-utilities";
+
+.pf-c-empty-state {
+  --pf-c-empty-state--MaxWidth: #{pf-size-prem(400px)}; // one off to match design
+  --pf-c-empty-state--Padding: var(--pf-global--spacer--xl);
+  --pf-c-empty-state__icon--MarginBottom: var(--pf-global--spacer--lg);
+  --pf-c-empty-state__icon--FontSize: #{pf-size-prem(54px)}; // one off to match design
```

Comment:
is this the only time we modify the size of an icon? Do we need utility classes for this?

Reply:
I asked @mceledonia and he said this is a one-off. If we end up doing it again, we can create something to handle a big icon?

Reply:
👍